### PR TITLE
Builder: Bump MF2 version to 2024-01-24

### PR DESCRIPTION
**Pull request by VladislavsPerkanuks**

### CODE REVIEW

File: builder_test.go [hunk 1]
```diff
@@ -145,13 +136,58 @@ func Test_Builder(t *testing.T) {
...
+				).
+				Text(" something ").
+				CloseMarkup(
+					"close",
+					EmptyAttribute("empty1"),
+					VarAttribute("attr1", "var"),
+				).
+				SelfCloseMarkup(
+					"selfClosing",
+					LiteralAttribute("attr1", "༼ つ ◕_◕ ༽つ"),
```
Comment (mvilks) [inline builder_test.go:170]: 😄

File: builder_test.go [hunk 2]
```diff
@@ -145,13 +136,58 @@ func Test_Builder(t *testing.T) {
 				Spacing(""),
 			".match{$i}{$j}\n1 2{{\\{first\\}}}\n2 0{{second {$i}}}\n3 0{{{|\\\\a\\||}}}\n* *{{{1}}}",
 		},
+		{
+			"attributes",
+			NewBuilder().
+				Text("Attributes for variable expression ").
+				Expr(
+					Var("i").
```
Comment (klavinsjanis) [inline builder_test.go:144]: Why not variadic attributes for method Var()?

File: builder_test.go [hunk 3]
```diff
@@ -145,13 +136,58 @@ func Test_Builder(t *testing.T) {
...
+					LiteralAttribute("attr1", "༼ つ ◕_◕ ༽つ"),
+				).
+				// nested markup
+				OpenMarkup("nest1").
+				OpenMarkup("nest2").
+				Text("nested").
+				SelfCloseMarkup("nest3").
+				CloseMarkup("nest2").
+				CloseMarkup("nest1"),
+			"{ #open opt1 = val1 opt2 = $var @attr1 = 1 } something { /close @empty1 @attr1 = $var }{ #selfClosing @attr1 = |༼ つ ◕_◕ ༽つ| / }{ #nest1 }{ #nest2 }nested{ #nest3 / }{ /nest2 }{ /nest1 }",
```
Comment (mvilks) [inline builder_test.go:179]: Is there a whitespace after "/" in self-closing markup?
`markup       = "{" [s] markup-open *(s attribute) [s] ["/"] "}"`

File: builder_test.go (hunk 1 above)
Comment (VladislavsPerkanuks) [reply builder_test.go:170]: :)

File: builder_test.go (hunk 2 above)
Comment (VladislavsPerkanuks) [reply builder_test.go:144]: 1. Only one variable is possible
2. Not this PR scope

File: builder_test.go (hunk 3 above)
Comment (VladislavsPerkanuks) [reply builder_test.go:179]: yup, good catch

**Review by DarjaLeonova — APPROVED** [review]

**Review by mvilks — APPROVED** [review]

**Review by klavinsjanis — APPROVED** [review]

**Pull request merged**